.github/workflows/end2end.yaml

```diff
@@ -383,166 +383,6 @@ jobs:
           cache-from: type=gha,scope=end2end-ctst
           cache-to: type=gha,mode=max,scope=end2end-ctst
 
```

Comment:
why are you removing the e2e (non ctst) tests?
these tests are not currently covered by CTST, must be preserved

Reply:
The PR is not finalized, I want to pass the CIs first with the latest images, and get reviews, then I will remove the left overs to reduce number of CI builds, so yes, they won't be removed

Reply:
you should re-enable them already, now that your tests are somewhat finalized : so we may detect already if there are some unexpected impact on backbeat...

Reply:
Done